package.json

```diff
@@ -22,7 +22,9 @@
     "test": "jest --config jest.test.config.js",
     "test:watch": "jest --config jest.test.config.js --watch",
     "playground:start": "cd playground; yarn start",
-    "playground:build": "cd playground; yarn build"
+    "playground:build": "cd playground; yarn build:playground",
```

Comment:
We can use yarn‘s build in —cwd if I am not mistaken. Alternatively pushd and popd could be nice just to be able to go back again. Not even sure we ever have to. 😇

Reply:
True, I just double checked. It's possible to use `--cwd` 💯